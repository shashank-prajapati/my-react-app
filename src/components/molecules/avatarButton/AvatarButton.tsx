import { Box } from "@mui/material";
import React from "react";
import AccountAvatar from "../../atoms/accountAvatar/AccountAvatar";
import ExpandMoreLogo from "../../atoms/expandMoreIcon/ExpandMoreLogo";

type Props = {};

function AvatarButton({}: Props) {
  return (
    <Box
      sx={{
        display: "flex",
        alignItems: "center",
      }}
    >
      <AccountAvatar />
      <ExpandMoreLogo />
    </Box>
  );
}

export default AvatarButton;
