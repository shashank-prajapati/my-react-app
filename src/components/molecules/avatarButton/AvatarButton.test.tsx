import * as React from 'react';
import { render, fireEvent } from '@testing-library/react';
import AvatarButton from './AvatarButton';
import { Typography } from '@mui/material';


describe('Testing ImageAtom', ()=>{
    it('should match the default snapshot',()=>{
        const {container} = render(<AvatarButton/>,);
        expect(container).toMatchSnapshot();
    })
})

