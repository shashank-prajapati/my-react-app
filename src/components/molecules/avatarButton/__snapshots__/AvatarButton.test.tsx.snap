// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing ImageAtom should match the default snapshot 1`] = `
<div>
  <div>
    <div
      class="MuiBox-root css-1srz2ck"
    >
      <div>
        <div
          class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-x4blui-MuiAvatar-root"
        >
          A
        </div>
      </div>
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-198kmue-MuiSvgIcon-root"
        focusable="false"
      >
        expandMore.svg
      </svg>
    </div>
  </div>
</div>
`;
